### DISCUSSION TITLE
Upload error

### DISCUSSION THREAD

**Discussion by PeterButler41** [Cookie-AutoDelete/Cookie-AutoDelete#1282] [Q&A / Support] [answered]
What is wrong with the attached .json file? I had to change file type to .txt
[CAD_Expressions_2021-12-18_17.22.32.txt](https://github.com/Cookie-AutoDelete/Cookie-AutoDelete/files/7740262/CAD_Expressions_2021-12-18_17.22.32.txt)

**Comment by kennethtran93** [comment, marked as answer]
According to the file you're missing the end array `]` near the end (between line 100 and 101).
![image](https://user-images.githubusercontent.com/6724477/146662331-ed75fa13-26f8-42dd-a423-f28d80832894.png)
should be this (line 101):
![image](https://user-images.githubusercontent.com/6724477/146662347-40f190a5-e5ce-4a1e-b995-9826c4a9b8d5.png)

**Reply by PeterButler41** [reply]
Thank you. The error message gave a wrong byte offset. I might have found it myself with a better byte number. Line number would have been better.

**Reply by kennethtran93** [reply]
> Thank you. The error message gave a wrong byte offset. I might have found it myself with a better byte number. Line number would have been better.

You're welcome.  Those error messages are by JavaScript/json themselves.  We don't do the json validations - I let the browser/JavaScript check it over first and then do my checks when it is valid.
